<ui version="4.0" >
 <class>ScheduleSMS</class>
 <widget class="QMainWindow" name="ScheduleSMS" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>448</width>
    <height>293</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>SendScheduleSMS</string>
  </property>
  <property name="styleSheet" >
   <string>QMainWindow {
 
background-color:rgb(246,243,243);

}</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QTreeWidget" name="view" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>30</y>
      <width>191</width>
      <height>291</height>
     </rect>
    </property>
    <property name="selectionMode" >
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <column>
     <property name="text" >
      <string>1</string>
     </property>
    </column>
   </widget>
   <widget class="QTextEdit" name="MsgEdit" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>30</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Maximum" hsizetype="Expanding" >
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize" >
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="tabChangesFocus" >
     <bool>false</bool>
    </property>
    <property name="acceptRichText" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="TB_Send" >
    <property name="geometry" >
     <rect>
      <x>360</x>
      <y>330</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>SendSMS</string>
    </property>
   </widget>
   <widget class="QLabel" name="input_nm" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>10</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="MsgBrowser" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>240</y>
      <width>251</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line" >
    <property name="geometry" >
     <rect>
      <x>190</x>
      <y>135</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lb_choseUser" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>180</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>one Sms at one time</string>
    </property>
   </widget>
   <widget class="QLabel" name="Tiem_scope" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>150</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>time scope</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>210</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="displayFormat" >
     <string>M/d/yyyy</string>
    </property>
    <property name="date" >
     <date>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit" >
    <property name="geometry" >
     <rect>
      <x>340</x>
      <y>210</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="displayFormat" >
     <string>h:mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="Schedule_Sms_Manage" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>350</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="palette" >
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="font" >
     <font/>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text" >
     <string>&lt;a href="http://www.libfetion.cn">Schedule Sms Manage&lt;/a></string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>300</x>
      <y>210</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>400</x>
      <y>210</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>H:M</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
